CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/importing/ExtractWordsFromTextPluginNGTest.java

```diff
@@ -549,4 +585,35 @@ public void testExtractWordFromContentRegexOnlyLowerCaseFalseParameter() throws
         assertTrue(newNodes.contains("do"));
         assertTrue(newNodes.contains("DO"));
     }
+
+    /**
+     * Test updateParameters
+     *
+     */
+    @Test
+    public void testUpdateParameters() {
+        final int vertexIdentifierAttributeId = VisualConcept.VertexAttribute.IDENTIFIER.ensure(graph);
+        final int transactionContentAttributeId = ContentConcept.TransactionAttribute.CONTENT.ensure(graph);
+        final int transactionTypeAttributeId = AnalyticConcept.TransactionAttribute.TYPE.ensure(graph);
+
+        graph.getSchema().newGraph(graph);
+        final int srcId = graph.addVertex();
+        graph.setStringValue(vertexIdentifierAttributeId, srcId, "Node0");
+
+        final int dstId = graph.addVertex();
+        graph.setStringValue(vertexIdentifierAttributeId, dstId, "Node1");
+
+        final int txId = graph.addTransaction(srcId, dstId, true);
+        graph.setObjectValue(transactionTypeAttributeId, txId, AnalyticConcept.TransactionType.COMMUNICATION);
+        graph.setStringValue(transactionContentAttributeId, txId, "Can you do this?\nWill you  DO  that?");
+
+        ExtractWordsFromTextPlugin instance = new ExtractWordsFromTextPlugin();
+        Graph graph1 = new DualGraph(graph.getSchema(), graph);
+
+        PluginParameters parameters = instance.createParameters();
+        parameters.getParameters().get(ExtractWordsFromTextPlugin.ATTRIBUTE_PARAMETER_ID).setStringValue(ContentConcept.TransactionAttribute.CONTENT.getName());
+
+        instance.updateParameters(graph1, parameters);
+        assertTrue(parameters.hasParameter(ExtractWordsFromTextPlugin.ATTRIBUTE_PARAMETER_ID));
```

Comment:
Having had a quick look at the function being tested here, I think it would be good to have some asserts confirming the function worked as intended. Currently, I don't think the assert here is adding anything as the function doesn't appear to be adding or removing that parameter (so it will always be true)